src/platforms/stm32/src/lib/sys.c

```diff
@@ -144,7 +144,7 @@ bool sys_lock_pin(GlobalContext *glb, uint32_t gpio_bank, uint16_t pin_num)
     }
 
     struct LockedPin *data = malloc(sizeof(struct LockedPin));
-    if (IS_NULL_PTR(data)) {
+    if (UNLIKELY(IS_NULL_PTR(data))) {
```

Comment:
I believe `UNLIKELY(IS_NULL_PTR(...))` is redundant so the UNLIKELY bit can be removed

Reply:
You are right, I forgot this macro already had branching hints. Fixed.